.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: clojure
+lein: lein2
+jdk:
+  - oraclejdk7
+  - openjdk7
```

Comment:
We've been including oraclejdk8 elsewhere ... seems like a good idea to do that here too?